components/cli/pkg/commands/project/init.go

```diff
@@ -108,27 +163,170 @@ func RunInit(cli cli.Cli, projectName string) error {
 					},
 				},
 			},
-		}); err != nil {
-			return fmt.Errorf("error occurred while initializing the project, %v", err)
+		})
+		if err != nil {
+			util.ExitWithErrorMessage("Error occurred while initializing the project", err)
```

Comment:
Return the error instead of exiting, in all places.

Reply:
Fixed